From: Andrea Bolognani <eof@kiyuko.org>
Date: Mon, 9 Oct 2023 23:17:10 +0200
Subject: meson: Fix XDR check for GNU/Hurd

The situation is the same as Linux: since glibc no
longer includes the RPC functionality, libtirpc must
be used to complement it.
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 1529cea..c0f4e1c 100644
--- a/meson.build
+++ b/meson.build
@@ -867,7 +867,7 @@ if not get_option('driver_remote').disabled()
   # use extra library as it's provided by libc directly.
   if host_machine.system() == 'windows'
     xdr_dep = cc.find_library('portablexdr', required: get_option('driver_remote'))
-  elif host_machine.system() == 'linux'
+  elif host_machine.system() in [ 'linux', 'gnu' ]
     xdr_dep = dependency('libtirpc', required: get_option('driver_remote'))
   elif host_machine.system() in [ 'freebsd', 'darwin' ]
     xdr_dep = cc.find_library('c', required: get_option('driver_remote'))
